src/coreclr/scripts/superpmi.py

```diff
@@ -297,6 +301,7 @@
 replay_common_parser.add_argument("--force_download", action="store_true", help=force_download_help)
 replay_common_parser.add_argument("-jit_ee_version", help=jit_ee_version_help)
 replay_common_parser.add_argument("-private_store", action="append", help=private_store_help)
+replay_common_parser.add_argument("-compile", help=compile_help)
```

Comment:
Should we accept -c as well here? I believe
`replay_common_parser.add_argument(["-compile", "-c"], ...)`
should do that. What's your opinion @BruceForstall?

Reply:
We can do this: 
`replay_common_parser.add_argument("-c", "--compile", help=compile_help)`
And then we can use the short and long version of the argument

Reply:
I like the idea of accepting just `-c` also. We kind of have a parameter naming convention here that arguments that take values have a single leading dash, so should it be:
```
replay_common_parser.add_argument("-c", "-compile", help=compile_help)
```
(or what Jakob suggested)?

Reply:
I have implemented this @BruceForstall